airflow/providers/airbyte/CHANGELOG.rst

```diff
@@ -24,6 +24,25 @@
 Changelog
 ---------
 
+3.2.0
+.....
+
+This release of provider is only available for Airflow 2.3+ as explained in the Apache Airflow
+providers support policy https://github.com/apache/airflow/blob/main/README.md#support-for-providers
```

Comment:
```suggestion
This release of provider is only available for Airflow 2.3+ as explained in the
`Apache Airflow providers support policy <https://github.com/apache/airflow/blob/main/README.md#support-for-providers>`_.
```

Probably better if we use a link here?

Reply:
Right. I am hesitant in putting any markup in CHANGELOG - because it becomes part of the README.rst and eventually becomes part of the package in PyPI - but links are save and they should both render correctly and be accepted by PyPI (we will see if they are passing our CI - we have `twine` check as part of our CI and they will fail if the README's are not complying with `PyPI` expectations.

Reply:
Updated - also for past changelog entries.